Tools/WebKitTestRunner/libwpe/EventSenderProxyLibWPE.cpp

```diff
@@ -48,7 +48,7 @@ EventSenderProxy::EventSenderProxy(TestController* testController)
     , m_clickButton(kWKEventMouseButtonNoButton)
 {
 #if PLATFORM(WPE) && ENABLE(WPE_PLATFORM)
-    if (testController->useWPEPlatformAPI())
+    if (!testController->useWPELegacyAPI())
         m_client = makeUnique<EventSenderProxyClientWPE>(*testController);
     else
         m_client = makeUnique<EventSenderProxyClientLibWPE>(*testController);
```

Comment:
While this diff keeps the change to a minimum, I think it is better to avoid negated clauses when it's possible  (you can swap the if / else blocks).